### PR TITLE
OSDOCS-8389 rosaworkshop.io Lab overview section

### PR DESCRIPTION
**Pull request by xJustin**
Port of Lab overview for rosaworkshop.io

Version(s):

4.13
Issue:

https://issues.redhat.com/browse/OSDOCS-8389

Link to docs preview:

https://70667--ocpdocs-pr.netlify.app/openshift-rosa/latest/cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-lab-overview
QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-lab-overview.adoc [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 22-JAN-2024
+
+== Lab overview
+
+=== Lab resources
+
+* source code for the OSToy application: link:https://github.com/openshift-cs/ostoy[]
```
Comment (EricPonvelle) [inline cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-lab-overview.adoc:16]: s/source/Source code...

I would also put the text into the link boxes so they are linked text.

File: cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-lab-overview.adoc [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+* Increase the load to view automatic scaling of the pods to handle the load using the Horizontal Pod Autoscaler.
+* Optional: Connect to an AWS S3 bucket to read and write objects.
+
+### OSToy Application Diagram
+
+image::ostoy-arch.png[OSToy architecture diagram]
+
+### Understanding the OSToy UI
+
+image::ostoy-homepage.png[Preview of the OSToy homepage]
```
Comment (EricPonvelle) [inline cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-lab-overview.adoc:262]: This image will need updates in the future.

**Review by EricPonvelle — COMMENTED** [review]
Just some small things you may want to correct + a capitalization thing

File: cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-lab-overview.adoc (hunk 1 above)
Comment (xJustin) [reply cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-lab-overview.adoc]: Updated!

**Pull request merged**